src/rectangle.js

```diff
@@ -36,42 +36,63 @@
 
 /**
  * @class Rect
- * @classdesc A Rectangle really represents a 2x2 matrix where each row represents a
- * 2 dimensional vector component, the first is (x,y) and the second is
- * (width, height).  The latter component implies the equation of a simple
- * plane.
+ * @classdesc A Rectangle is described by it top left coordinates (x, y), width,
+ * height and degrees of rotation around (x, y).
+ * Note that the coordinate system used is the one commonly used with images:
+ * x increases when going to the right
+ * y increases when going to the bottom
+ * degrees increases clockwise with 0 being the horizontal
  *
  * @memberof OpenSeadragon
- * @param {Number} x The vector component 'x'.
- * @param {Number} y The vector component 'y'.
- * @param {Number} width The vector component 'height'.
- * @param {Number} height The vector component 'width'.
+ * @param {Object} options
```

Comment:
I'm not in favor of moving to an `options`-based constructor for Rect; it's too basic of a component, and the standard set of arguments is already very familiar from other libraries. Adding degrees onto the end does make it a little longer (and is less of a standard), but it's not enough to make me want to go all the way to an `options`-based constructor. 

I guess I'd be fine with adding the options option, but I don't want to deprecate the other form.